test/charts/utilization-bar/utilization-bar.spec.js

```diff
@@ -32,6 +36,18 @@ describe('Directive: pfUtilizationBarChart', function() {
     return el;
   };
 
+  it("should set value of data.subdata length to be 3", function() {
+    $scope.layout.type = 'multidata';
+    $scope.data = {
+      'total': '100',
+      'subdata' : [ { "used" : 45 , "color" : "#00558a" , "subtitle" : "Object" },
+                    { "used" : 15 , "color" : "#0071a6" , "subtitle" : "Block" },
+                    { "used" :  5 , "color" : "#00a8e1" , "subtitle" : "OpenStack" }]
+    };
```

Comment:
Please choose colors from the Patternfly color palette -  [Color Palette](https://www.patternfly.org/styles/color-palette/)